src/c-unchop.c

```diff
@@ -104,6 +104,7 @@ r_obj* list_unchop(r_obj* xs,
   const struct vec_proxy_assign_opts unchop_proxy_assign_opts = {
     .ownership = VCTRS_OWNERSHIP_deep,
     .recursively_proxied = true,
+    .style = VCTRS_ASSIGNMENT_STYLE_location,
```

Comment:
This will eventually be an argument to `list_unchop()`, well `list_multi_assign()` really